## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 426922`
Commit: `f8254ca555e05cd797911c0751ca214660de1676`

---
### `x86_64-linux`
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>python312Packages.bambi</li>
    <li>python312Packages.bambi.dist</li>
    <li>python312Packages.nutpie</li>
    <li>python312Packages.nutpie.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 2 packages built:</summary>
  <ul>
    <li>python312Packages.pymc</li>
    <li>python312Packages.pymc.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:x: 6 packages failed to build:</summary>
  <ul>
    <li>python312Packages.bambi</li>
    <li>python312Packages.bambi.dist</li>
    <li>python312Packages.nutpie</li>
    <li>python312Packages.nutpie.dist</li>
    <li>python312Packages.pymc</li>
    <li>python312Packages.pymc.dist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>python312Packages.bambi</li>
    <li>python312Packages.bambi.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 4 packages built:</summary>
  <ul>
    <li>python312Packages.nutpie</li>
    <li>python312Packages.nutpie.dist</li>
    <li>python312Packages.pymc</li>
    <li>python312Packages.pymc.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>python312Packages.bambi</summary>
<pre>adding 'bambi/terms/common.py'
adding 'bambi/terms/group_specific.py'
adding 'bambi/terms/hsgp.py'
adding 'bambi/terms/offset.py'
adding 'bambi/terms/response.py'
adding 'bambi/terms/utils.py'
adding 'bambi-0.15.0.dist-info/licenses/LICENSE'
adding 'bambi-0.15.0.dist-info/METADATA'
adding 'bambi-0.15.0.dist-info/WHEEL'
adding 'bambi-0.15.0.dist-info/top_level.txt'
adding 'bambi-0.15.0.dist-info/RECORD'
removing build/bdist.linux-x86_64/wheel
Successfully built bambi-0.15.0-py3-none-any.whl
Finished creating a wheel...
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
@nix { "action": "setPhase", "phase": "pythonRuntimeDepsCheckHook" }
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for bambi-0.15.0-py3-none-any.whl
  - pymc>=5.18.0 not satisfied by version 0+unknown</pre>
</details>
<details>
<summary>python312Packages.nutpie</summary>
<pre>         0.42383589, 0.35015801, 0.38888445, 1.39506569, 0.72770102,
         1.80674255, 0.46638885, 1.6157389 , 1.6157389 , 0.42773956,
         0.21798299, 0.14578963, 1.0132065 , 1.0132065 , 1.19713289,
         0.39079146, 0.22368685, 0.14901917, 0.10386585, 0.1537675 ,
         0.12941973, 0.34637114, 0.81455337, 2.41042419, 0.42738952,
         0.32229096, 0.24891091, 0.85440412, 1.35372438, 1.35372438,
         2.0054608 , 0.04578806, 0.04156436, 0.07975505, 0.09130765,
         0.07094797, 0.00993872, 0.42144829, 0.55037687, 0.60938676,
         0.49048681, 2.6607037 , 0.32803965, 0.38599937, 0.49729386,
         1.67108574, 1.14327509, 1.14327509, 0.90306264, 0.90306264,
         0.90306264, 0.69126881, 2.00150821, 0.58767151, 0.79678972,
         1.35562904, 0.59847138, 0.68182582, 0.81829614, 1.14264835,
         0.11309381, 0.25086143, 0.28449118, 0.00420445, 0.00566936]), which will be an error in a future version of pytest.  Did you mean to use `assert` instead of `return`?
    warnings.warn(

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/test_pymc.py::test_wait_timeout[numba-None] - Failed: Timeout (>20.0s) from pytest-timeout.
FAILED tests/test_pymc.py::test_pause[numba-None] - Failed: Timeout (>20.0s) from pytest-timeout.
======= 2 failed, 54 passed, 2 xpassed, 7 warnings in 1362.24s (0:22:42) =======</pre>
</details>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>python312Packages.bambi</summary>
<pre>adding 'bambi/terms/base.py'
adding 'bambi/terms/common.py'
adding 'bambi/terms/group_specific.py'
adding 'bambi/terms/hsgp.py'
adding 'bambi/terms/offset.py'
adding 'bambi/terms/response.py'
adding 'bambi/terms/utils.py'
adding 'bambi-0.15.0.dist-info/licenses/LICENSE'
adding 'bambi-0.15.0.dist-info/METADATA'
adding 'bambi-0.15.0.dist-info/WHEEL'
adding 'bambi-0.15.0.dist-info/top_level.txt'
adding 'bambi-0.15.0.dist-info/RECORD'
removing build/bdist.macosx-11.3-arm64/wheel
Successfully built bambi-0.15.0-py3-none-any.whl
Finished creating a wheel...
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for bambi-0.15.0-py3-none-any.whl
  - pymc>=5.18.0 not satisfied by version 0+unknown</pre>
</details>
</details>

--- a/pkgs/development/python-modules/pymc/default.nix
+++ b/pkgs/development/python-modules/pymc/default.nix
@@ -22,14 +22,14 @@
 
 buildPythonPackage rec {
   pname = "pymc";
-  version = "5.24.0";
+  version = "5.24.1";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "pymc-devs";
     repo = "pymc";
     tag = "v${version}";
-    hash = "sha256-B4HFb+2Hzxt/eK5PdE9wNxQRNouSPi/9aKSrBV8xba4=";
+    hash = "sha256-lBsVr3A7y9CP+41Tk5TraWhu59HJb2+r0I7WlXiP/sY=";
   };
 
   build-system = [

